### ISSUE TITLE
Unable to start master-4.1 

### ISSUE THREAD

**Issue by bryanboza** [dotCMS/core#11182]
After the last changes we are unable to start master-4.1, we are having the follow error and the system isn't starting: 
```
[30/03/17 12:13:18:539 CST] ERROR util.JDBCExceptionReporter: Cannot add or update a child row: a foreign key constraint fails ("sender"."contentlet", CONSTRAINT "fk_user_contentlet" FOREIGN KEY ("mod_user") REFERENCES "user_" ("userid"))
[30/03/17 12:13:18:539 CST] ERROR impl.SessionImpl: Could not synchronize database state with session 
```

**FYI:** I tried multiple times with a fresh install and multiple environments and databases and have the same problem every time 

Here the complete log: https://gist.github.com/bryanboza/d0b754a752c3a968bbb786f3f71bbf0f  

## Your Environment
* dotCMS version used: Master-4.1 
* Browser Name and version: FF 
* Operating System and version: MacOSX 
* Application Server and version: Tomcat 8 
* Java Brand and version: Java 8
* Database and version: MySQL

**Comment by bryanboza** [comment]
We still having problems in h2 https://gist.github.com/bryanboza/d25ca7a995912ae4a6e225200ef7b4e8

**Comment by jgambarios** [comment]
We should be ok now.

**Comment by bryanboza** [comment]
Fixed, now we can start master-4.1 without problems
